staging/src/k8s.io/apiserver/pkg/storage/cacher/watch_cache.go

```diff
@@ -301,6 +310,7 @@ func (w *watchCache) processEvent(event watch.Event, resourceVersion uint64, upd
 
 // Assumes that lock is already held for write.
 func (w *watchCache) updateCache(event *watchCacheEvent) {
+	w.resizeCache(event)
```

Comment:
Let's pass event.RecordTime down the function maybe? It doesn't need the whole event - and the signature will be cleaner.

Reply:
addressed.